src/components/structures/TimelinePanel.js

```diff
@@ -191,6 +189,12 @@ const TimelinePanel = createReactClass({
 
             // always show timestamps on event tiles?
             alwaysShowTimestamps: SettingsStore.getValue("alwaysShowTimestamps"),
+
+            // how long to show the RM for when it's visible in the window
+            readMarkerInViewThresholdMs: SettingsStore.getValue("readMarkerInViewThresholdMs"),
+
+            // how long to show the RM for when it's scrolled off-screen
+            readMarkerOutOfViewThresholdMs: SettingsStore.getValue("readMarkerOutOfViewThresholdMs"),
```

Comment:
these won't update without switching between rooms

Reply:
correct. does i wonder if this matters though? it's what we do for 12h timestamps etc too.

Reply:
well, we should be better at these sorts of things. There's bugs all over the place for "X does not update after changing the setting". They weren't problems before because the timeline would remount after the settings page went away, but now we're a dialog and that causes a whole set of problems.